readme.md

```diff
@@ -0,0 +1,176 @@
+# Tech Spec - Teleport Challenge
+
+## Overview
+To be implemented:
+
+1. `Job` library for starting, stopping, and getting status and streamed logs of jobs.
+2. API that implements the `Job` library, permitting clients to call the 4 public methods.
+3. CLI Client that calls API, using command line args to specify API methods.
+
+## Job Library
+`Job` type will include public fields:
+
+- Command
+- Args
+- ProcessID
+
+##### Implement methods for:
+
+- Start
+	- Assure job command exists. Executables can be located in Go using `exec.LookPath()`.
```

Comment:
This check is uneeded, any error will be returned when trying to start a job. This check furthermore does not have any value because it can race due to the fact that the FS can be modified inbetween this check and command execution.